native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/serializer/GlueSchemaRegistryKafkaSerializerTests.cs

```diff
@@ -117,17 +137,37 @@ public void KafkaSerDeTestForAllProtobufTypes(IMessage message)
             Assert.AreEqual(message, deserializedObject);
         }
 
+        [Test]
+        public void KafkaSerDeTestForJsonMessage()
+        {
+            var message = JsonDataWithSchema.Build(GENERIC_TEST_RECORD.Schema, GENERIC_TEST_RECORD.Payload);
+            var configs = new Dictionary<string, dynamic>
+            {
+                { GlueSchemaRegistryConstants.DataFormatType, GlueSchemaRegistryConstants.DataFormat.JSON },
+            };
+
+            KafkaSerializer.Configure(configs);
+            KafkaDeserializer.Configure(configs);
+
+            var serialized = KafkaSerializer.Serialize(message, "test-topic-json");
+            var deserializedObject = KafkaDeserializer.Deserialize("test-topic-json", serialized);
```

Comment:
Also assert the `getSchema` and `canDecode` methods. Let's cover all the methods.